demo/database/initdb/0_init.sql

```diff
@@ -0,0 +1,47 @@
+CREATE EXTENSION IF NOT EXISTS pg_trgm;
+
+CREATE TABLE book (
+    id serial PRIMARY KEY,
+    title text,
+    price bigint,
+    available boolean
+);
+
+-- Mocking dataset
+INSERT INTO book (title, price, available)
+    SELECT
+        left(md5(random()::text), 3+(17*random())::integer),
+        (random() * 100)::bigint,
+        (random() > 0.8)
+    FROM generate_series(1, 2000000);
+
+-- Initial indexes
+CREATE INDEX book_price_idx
+    ON book (price);
+
+CREATE INDEX book_available_title_idx
+    ON book USING gin (title gin_trgm_ops);
+
+CREATE INDEX book_available_idx
+    ON book (available);
+
+-- Candidate indexes
+CREATE INDEX book_price_available_partial
+    ON book (price) where available IS true;
+
+CREATE INDEX book_price_available_is_expr_idx
+    ON book (price, available);
+
+CREATE INDEX book_available_is_expr_price_idx
+    ON book (available, price);
+
+CREATE INDEX book_title_idx
+    ON book (title);
+
+-- Adding entropy
+UPDATE book SET price = price + 1 WHERE price % 10 = 7;
+UPDATE book SET price = price - 1 WHERE price % 10 = 8;
```

Comment:
I'm not sure we need to do this. Is it really the subject of this library? I think we want to compare indexes no matter their bloat state, no?